seolondon/settings/__init__.py

```diff
@@ -197,6 +200,10 @@
             'PORT': os.environ['RDS_PORT'],
         }
     }
+elif 'DATABASE_URL' in os.environ:
```

Comment:
Same here this is set in production settings already, why is it needed here as well?